src/settings.js

```diff
@@ -0,0 +1,46 @@
+const SettingsClient = require('./settingsClient')
+const Promise = require('promise')
+
+/**
+ * @class SettingsValues
+ * @classdesc
+ * <b><i> The SettingsValues class provides access to all settings values of the (most of the time authenticated) user.</i></b>
+ *
+ * @author Noveen Sachdeva
+ * @version 1.0.0
+ * @param {object}  helperFile  instance of the helpers class
+ */
+class SettingsValues {
+  constructor (helpers) {
+    this.helpers = helpers
+  }
+
+  /**
+   * Gets all settings values for the given account uuid. If no uuid is provided, settings values
+   * for the authenticated user will be fetched.
+   * @returns {Promise.<settings>} array: all available settings values
+   * @returns {Promise.<error>}    string: error message, if any.
+   */
+  async getSettingsValues (accountUuid = 'me') {
```

Comment:
is "me" the default ? if yes please add in the JSDoc that "me" will be used as default and also what it means (probably the user's profile?)

Reply:
Yes, `me` is the default. Added param and description to the docblock.